### DISCUSSION TITLE
I need to add a different layout for grouped and ungrouped routes.

### DISCUSSION THREAD

**Discussion by MSR-07** [vercel/next.js#61706] [Help]
### Summary

I would like to implement different layouts for the auth route and other routes in my app by giving it a routing structure. Can you assist me with this?
![Screenshot 2024-02-06 134648](https://github.com/vercel/next.js/assets/102695412/f9fd9769-f5a7-4d1d-9d22-6eb25298bd08)

**Comment by makibat** [comment]
You could consider using the following structure:

```
├── app
│   ├── (main) // Group all other routes
│   │   ├── ...
│   │   └── layout.tsx // Layout used on every route other than /auth
│   ├── auth
│   │   ├── page.tsx
│   │   └── layout.tsx // Unique layout only used on /auth
│   └── layout.tsx // Root layout which contains <html />, <body /> and {children}
└── ...
```

**Reply by MSR-07** [reply]
I want to make it clear that I do not want the root layout to appear on my authentication page.

**Reply by makibat** [reply]
Okay, sounds like you just have to remove the top-level `layout.tsx` file and that way you'll have multiple root layouts: 
one for auth and one for other routes.

**Reply by MSR-07** [reply]
You are right......
